src/android/OnLocationResultEventListener.java

```diff
@@ -3,8 +3,7 @@
 import com.google.android.gms.location.LocationResult;
 
 interface OnLocationResultEventListener {
-
     void onLocationResultSuccess(LocationContext locationContext, LocationResult result);
```

Comment:
Since there's a new `onLocationResultSuccessHuawei` method, maybe this one should be changed to `onLocationResultSuccessGoogle`?